bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/coap/ShellyCoapHandler.java

```diff
@@ -557,12 +554,12 @@ private void handleStatusUpdate(String devId, String payload, int serial) {
             } catch (IllegalArgumentException | NullPointerException | ArrayIndexOutOfBoundsException e) {
                 // even the processing of one value failed we continue with the next one (sometimes this is caused by
                 // buggy formats provided by the device
-                logger.debug("{}: Unable to process data from sensor[{}]: Dev={}{}", thingName, i, devId, e.toString());
+                logger.debug("{}: Unable to process data from sensor[{}], devId={}: {}", thingName, i, devId,
+                        e.toString());
```

Comment:
If you want the stack trace, the proper way to do it is like this:
```suggestion
                logger.debug("{}: Unable to process data from sensor[{}], devId={}:", thingName, i, devId,
                        e);
```

Reply:
that's what I tried, but doesn't work, the code checker reports in incorrect number of arguments even if the statement has the correct number of args

Reply:
This change is correct.
```suggestion
                logger.debug("{}: Unable to process data from sensor[{}], devId={}:", thingName, i, devId,
                        e);
```

Reply:
changed